docs/CHANGELOG/changelog-0.9.x.md

```diff
@@ -1,79 +1,79 @@
 
-- Logs in all microservices have been changed to always log what is being executed, rather than only when the LOG_LEVEL is set to debug. [#1507](https://github.com/berops/claudie/pull/1507)
+- Logs in all microservices have been changed to always log what is being executed, rather than only when the LOG_LEVEL is set to debug. [#1507](https://github.com/berops/Claudie/pull/1507)
 
-- Longhron version was bumped from 1.6.0 to 1.7.0 [#1511](https://github.com/berops/claudie/pull/1511)
+- Longhron version was bumped from 1.6.0 to 1.7.0 [#1511](https://github.com/berops/Claudie/pull/1511)
 
-- When building a Kubernetes cluster without a load balancer for the API server, the generated kubeconfig will now work for all control plane nodes defined in the input manifest, instead of just one. [#1546](https://github.com/berops/claudie/pull/1546)
+- When building a Kubernetes cluster without a load balancer for the API server, the generated kubeconfig will now work for all control plane nodes defined in the input manifest, instead of just one. [#1546](https://github.com/berops/Claudie/pull/1546)
 
 
 ### Experimental
-- Support for a HTTP proxy was added. The HTTP Proxy can be turned on by setting the `HTTP_PROXY_MODE` environmanet variable in the claudie config map to `on`  [#1440](https://github.com/berops/claudie/pull/1440)
+- Support for a HTTP proxy was added. The HTTP Proxy can be turned on by setting the `HTTP_PROXY_MODE` environment variable in the Claudie config map to `on`  [#1440](https://github.com/berops/Claudie/pull/1440)
 
 ## Bug fixes
 - In the case when the infrastructure fails to be build or is only partially build
-  the deletion process was stuck on acquiring a lock which was never created, this issue is no longer present [#1463](https://github.com/berops/claudie/pull/1463)
-- The init process was added to the Ansible microservice because previously spawned Ansible playbooks left behind zombie processes that consumed resources. The init process takes care of cleaning up these processes. [#1527](https://github.com/berops/claudie/pull/1527)
-- Fixed an edge case where part of the load balancer infrastructure was incorrectly destroyed when a failure occurred in the middle of the workflow. [#1533](https://github.com/berops/claudie/pull/1533)
-- The whitespace when generating keys will no longer be trimmed [#1539](https://github.com/berops/claudie/pull/1539)
-- GenesisCloud autoscaling will now correctly work [#1543](https://github.com/berops/claudie/pull/1543)
\ No newline at end of file
+  the deletion process was stuck on acquiring a lock which was never created, this issue is no longer present [#1463](https://github.com/berops/Claudie/pull/1463)
+- The init process was added to the Ansible microservice because previously spawned Ansible playbooks left behind zombie processes that consumed resources. The init process takes care of cleaning up these processes. [#1527](https://github.com/berops/Claudie/pull/1527)
```

Comment:
```suggestion
- The init process was added to the Ansible microservice because previously spawned Ansible playbooks left behind zombie processes that consumed resources. The init process takes care of cleaning up these processes. [#1527](https://github.com/berops/claudie/pull/1527)
```